### DISCUSSION TITLE
Add icons to the tasks in the task list for more visual distinction among many items

### DISCUSSION THREAD

**Discussion by andergmartins** [publishpress/publishpress-checklists#432] [Feature Requests]
We started getting more available tasks in the Checklists page. It seems like it started became a little confuse.
I think if we add icons to each item according to its type we can make easier to distinguish and find each item.

A rough mockup:
<img width="1279" alt="Screen Shot 2020-07-27 at 12 50 45" src="https://user-images.githubusercontent.com/407131/88564487-a7fed400-d009-11ea-8362-7b035e2fb92a.png">

I'm still not sure we should display those icons in the checklist metabox (in the post edit pages) because we already have the check/unchecked mark.

**Comment by stevejburge** [comment]
That's a good idea for the settings. I had been thinking of some sub-headings, but this might be better

**Comment by andergmartins** [comment]
After implementing #162 we would have more room for the icon
